examples/scenarioSepMomentumManagement.py

```diff
@@ -578,6 +578,38 @@ def run(momentumManagement, cmEstimation, showPlots):
     # but it receives inputs and provides outputs to modules that run on the main flight software task
     messaging.VehicleConfigMsg_C_addAuthor(cmEstimator.vehConfigOutMsgC, vcMsg_CoM)
 
+    # # Uncomment to Enable Vizard
```

Comment:
We should execute as much of the scenario code in the CI tests as possible.  Use `vizInterface.vizFound` to guard against cases where this module is not build.  By default the scenario should not safe off the binary file.  Also, the custom textures should be available to the BSK script and not custom to a particular computer.  I pushed these changes as a commit to review and squash with this branch.